modules/standard/List.chpl

```diff
@@ -1731,6 +1737,25 @@ module List {
       _leave();
     }
 
+    pragma "no doc"
```

Comment:
I might switch this to `@chpldoc.nodoc` since the pragma is going to be (or maybe already is) deprecated.